packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue

```diff
@@ -11,9 +11,11 @@ export default {
 
 
 <style lang="scss" module>
+@use '@justeat/fozzie/src/scss/fozzie' as f;
+
 .c-heading {
-    height: $skeleton-loader-heading-height;
+    height: common.$skeleton-loader-heading-height;
```

Comment:
How is the `common` module brought in?

Reply:
@xander-marjoram it's imported via the vue.config.js file for each component. We added some docs here to explain:
https://github.com/justeat/fozzie-components#reusable-styling-with-commonscss

Reply:
Thanks, makes sense once you've read the webpack config!